contracts/libraries/mars-lending/src/msg.rs

```diff
@@ -0,0 +1,130 @@
+use cosmwasm_schema::{cw_serde, QueryResponses};
+use cosmwasm_std::{Addr, Deps, DepsMut, Uint128};
+use cw_ownable::cw_ownable_query;
+use valence_library_utils::LibraryAccountType;
+use valence_library_utils::{error::LibraryError, msg::LibraryConfigValidation};
+use valence_macros::{valence_library_query, ValenceLibraryInterface};
+
+#[cw_serde]
+pub enum FunctionMsgs {
+    /// Message to lend tokens.
+    Lend {},
+    /// Message to withdraw tokens. If amount is not specified, full amount will be withdrawn.
+    Withdraw { amount: Option<Uint128> },
+}
+
+#[valence_library_query]
+#[cw_ownable_query]
+#[cw_serde]
+#[derive(QueryResponses)]
+/// Enum representing the different query messages that can be sent.
+pub enum QueryMsg {}
+
+#[cw_serde]
+#[derive(ValenceLibraryInterface)]
+pub struct LibraryConfig {
+    // Address of the input account (Valence interchain account)
```

Comment:
```suggestion
    // Address of the input account
```

It's not an interchain account

Reply:
fixed 2724d5bb4178a4b0db589f54256aee8023e486e1